ts/components/wallet/WalletLayout.tsx

```diff
@@ -57,6 +57,7 @@ type Props = Readonly<{
   onNewPaymentPress?: () => void;
   allowGoBack: boolean;
   displayedWallets?: React.ReactNode;
+  backgroundColor?: StyleProp<ViewStyle>;
```

Comment:
This is not a backgroundColor this is the whole style.

Reply:
you're right. I updated its name as "contentStyle"